core/src/main/resources/AppConstants.properties

```
@@ -156,11 +156,11 @@ transactionmanager.btm.journal.maxRetries=500
 
 ## JMS Connection Pool properties for Narayana
 # Maximum number of physical connections that you can create in this pool.
-transactionmanager.narayana.jms.connection.maxPoolSize=5
+transactionmanager.narayana.jms.connection.maxPoolSize=10
 # Amount of time a connection can be unused or idle until it can be discarded.
 transactionmanager.narayana.jms.connection.maxIdleTime=60
 # Maximum number of jms sessions per connection that can be created in the connection pool.
-transactionmanager.narayana.jms.connection.maxSessions=500
+transactionmanager.narayana.jms.connection.maxSessions=80
```

```suggestion
transactionmanager.narayana.jms.connection.maxSessions=100
```
Misschien dat wij ook maar 100 moeten gebruiken als default, zie mijn commentaar bovenaan

---

It's fine, as a default. I agree.